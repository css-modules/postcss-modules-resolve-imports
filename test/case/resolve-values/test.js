'use strict';

const resolveImports = require('../../../index');
const setup = require('../../setup');

test('resolve-readme-example', () => {
  const {resulting, exports: tokens} = setup(
    'values',
    'extract-imports',
    resolveImports({icssExports: true})
  )(__dirname);

  expect(resulting).toMatchSnapshot();
  expect(tokens).toMatchSnapshot();
});
