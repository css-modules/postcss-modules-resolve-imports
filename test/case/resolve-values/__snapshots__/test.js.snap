// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolve-readme-example 1`] = `
":export
{
  green: #00FF00;
  localname: red;
}

/*  imported from node_modules/library/variable.css  */

/*  imported from source.css  */

.red
{
  color: #00FF00;
}
"
`;

exports[`resolve-readme-example 2`] = `
Object {
  "green": "#00FF00",
  "localname": "red",
}
`;
